crates/common/logged_command/src/logged_command.rs

```diff
@@ -227,21 +227,35 @@ impl LoggedCommand {
         match result.as_ref() {
             Ok(output) => {
                 if let Some(code) = &output.status.code() {
+                    let exit_code_msg = if code.eq(&0i32) { "OK" } else { "ERROR" };
                     logger
-                        .write_all(format!("exit status: {}\n\n", code).as_bytes())
+                        .write_all(
+                            format!("Exit status: {} ({exit_code_msg})\n\n", code).as_bytes(),
+                        )
                         .await?
                 };
                 if let Some(signal) = &output.status.signal() {
                     logger
-                        .write_all(format!("killed by signal: {}\n\n", signal).as_bytes())
+                        .write_all(format!("Killed by signal: {}\n\n", signal).as_bytes())
                         .await?
                 }
-                logger.write_all(b"stdout <<EOF\n").await?;
-                logger.write_all(&output.stdout).await?;
-                logger.write_all(b"EOF\n\n").await?;
-                logger.write_all(b"stderr <<EOF\n").await?;
-                logger.write_all(&output.stderr).await?;
-                logger.write_all(b"EOF\n").await?;
+                // Log stderr then stdout, so the flow reads chronologically
+                // as the stderr is used for log messages and the stdout is used for results
```

Comment:
I don't really see why having `stderr` before `stdout` is better. But I have no strong argument for the reverse.